packages/third-parties/src/google/gtm.tsx

```diff
@@ -31,6 +31,13 @@ export function GoogleTagManager(props: GTMParams) {
   const gtmAuth = auth ? `&gtm_auth=${auth}` : ''
   const gtmPreview = preview ? `&gtm_preview=${preview}&gtm_cookies_win=x` : ''
 
+  useEffect(() => {
+    // Useful for feature detection and measurement
+    performance.mark('next-third-parties', {
```

Comment:
Same here - though I like that we're doing a separate name for GTM. WIll help us track diff components and compare